docs/events/workshops/posts/2025-workshop.md

```diff
@@ -0,0 +1,72 @@
+---
+date: 2025-11-18
+---
+
+# G-ADOPT Workshop 2025
+
+The 2025 G-ADOPT workshop will be held at the
+[ANU's Research School of Earth Sciences](https://earthsciences.anu.edu.au/) in
+Canberra. Mark your diaries for **Tuesday 18/11/25 (9 am – 5 pm)**. We welcome all
+participants, including existing and prospective users, developers, and collaborators.
+
+The workshop will provide an opportunity for the G-ADOPT team to showcase recent
+development and advances on the forward and adjoint components of this finite-element
+modelling platform. Key areas of focus will include:
+
+1. Mantle and lithosphere dynamics;
+2. Glacial isostatic adjustment and visco-elasticity;
```

Comment:
I would drop 2 and 5 here, and write below that depending on the background and interests of registered participants, some aspects of GIA (and visco-elasticity) and Groundwater could be added to the program (or similar).